addon/helpers/to-money.js

```diff
@@ -1,9 +1,23 @@
-import Helper from 'ember-helper';
-import { isEmpty } from 'ember-utils';
-import toMoneyUtil from 'ember-cli-uniq/utils/to-money';
+import { isEmpty } from '@ember/utils';
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+import toMoney from 'ember-cli-uniq/utils/to-money';
 
-export function toMoney([params]) {
-  return isEmpty(params) ? '' : toMoneyUtil(params.amount, params.currency_code);
-}
+const DEFAULT_LOCALE = 'pt-pt';
```

Comment:
shouldn't the default be en-gb?

Reply:
It's debatable... Where most of our users come from? :trollface:

Reply:
But you're right! :)

Reply:
I know you did it in `pt-pt` to maintain what we have today, but since our default is always `en-gb`...